libs/pipelines/api_v2_pipeline.py

```diff
@@ -52,15 +51,16 @@ def latest(self) -> Dict[str, Any]:
 
     @property
     def timeseries(self) -> OneRegionTimeseriesDataset:
-        return self._combined_data.timeseries
+        return self._combined_data
 
     @staticmethod
     def from_region_and_model_output(
         region: pipeline.Region,
+        combined_data: MultiRegionTimeseriesDataset,
         rt_data: MultiRegionTimeseriesDataset,
         icu_data: MultiRegionTimeseriesDataset,
     ) -> "RegionalInput":
-        combined_data = combined_datasets.RegionalData.from_region(region)
+        one_region_data = combined_data.get_one_region(region)
```

Comment:
hehe this is a todo for the future, and controversial syntax, but could be cool if the syntax was `combined_data[region]`

Reply:
-1 for making slow operations hide behind simple syntax. I would be into making `MultiRegionTimeseriesDataset` objects iterable to yield fast access to all the regions.